boards/airbourne/airbourne_board.h

```diff
@@ -115,11 +121,13 @@ class AirbourneBoard : public Board
   void clock_delay(uint32_t milliseconds) override;
 
   // serial
-  void serial_init(uint32_t baud_rate) override;
+  void serial_init(uint32_t baud_rate, uint32_t dev) override;
   void serial_write(const uint8_t *src, size_t len) override;
   uint16_t serial_bytes_available() override;
   uint8_t serial_read() override;
   void serial_flush() override;
+  Serial** get_serial_interfaces(void);//Returns an array of pointers to available serial interfaces
+  uint8_t get_serial_count(void);//Returns the number of available serial interfaces. Same as length of array above.
```

Comment:
Are these functions used/do they have an implementation?

Reply:
They are legacy and need to be removed